src/groups/mqb/mqbblp/mqbblp_clusterqueuehelper.cpp

```diff
@@ -5861,7 +5861,8 @@ void ClusterQueueHelper::onCloseQueueResponse(
                   << contextSp->d_peer->nodeDescription();
 }
 
-void ClusterQueueHelper::gcExpiredQueues(bool immediate)
+void ClusterQueueHelper::gcExpiredQueues(mqbcmd::ClusterResult* result,
```

Comment:
The purpose of this was to provide an error response instead of silently dropping when running `CLUSTERS CLUSTER <name> FORCE_GC_QUEUES` on a non-primary. However, with routing this shouldn't be necessary anymore, so I'll get rid of it.

Reply:
I think it's a good practice to return false when something doesn't go right, regardless of admin routing and what not. Now it's silencing errors which is unideal. Better yet, make the function return an error code.

Reply:
I'll make it return an error code